### ISSUE TITLE
Attestation for GARUDA

### ISSUE THREAD

**Issue by p-j-l** [darobin/garuda#3]
In the [Objective section of GARUDA](https://darobin.github.io/garuda/#objective), there’s discussion of various ways to add trust to a server and it looks like there might be a technical way to add trust to one of the options.  This is a quote:

 > It's open source! That is great, and whatever Garuda produces will be open source, but just because something is open source doesn't prove that the system running it is actually running that open source implementation as is — it's pretty easy to cheat.

[Remote attestation](https://en.wikipedia.org/wiki/Trusted_Computing#Remote_attestation) is a feature of Trusted Execution Environments that can guarantee to a caller that a certain version of the software is running, [we’ve explored it a bit here](https://github.com/google/ads-privacy/tree/master/trust-model/tee).  It could work very nicely with Open Source software by giving browsers, or anyone, a way to guarantee that there’s no cheating by running other software so it might be worth including in the spec as a possibility.

**Comment by darobin** [comment]
Yes, that's very much in line with the kind of thing I would like to see happen. If we can rely on technical means to eliminate overhead and weak points (eg. the need to have actual on-site audits) then the governance structure can be kept to a minimum.
